nextjs-interface/src/lib/data.ts

```diff
@@ -39,7 +39,7 @@ export const fetchWithAuth = async (url: string) => {
       if (response.status === 401) {
         window.location.href = "/login";
       }
-      throw new Error(`HTTP error! Status: ${response.status}`);
+      new Error(`HTTP error! Status: ${response.status}`);
```

Comment:
Here you just don't show the error anymore... But the error is not resolved... I think